.github/workflows/tests.yml

```diff
@@ -386,27 +386,23 @@ jobs:
           CGO_ENABLED: 0
           GOARCH: amd64
         run: |
-          go build -o trimpath -o bin/lxc.x86_64 ./lxc
+          go build -ldflags "-s -w" -o trimpath -o bin/lxc.x86_64 ./lxc
```

Comment:
Are these used in snap build too?

Reply:
No, at the time, IIRC, I wasn't too sure how to fit those in for the CGO builds so I settled on strip'ing them afterward. Are you hinting me to go the `-ldflags "-s -w"` and `CGO_LDFLAGS` route for the snap and LXD too?

Reply:
Well I was thinking we could do them on the CLI build bits

Reply:
Do you mean those: https://github.com/canonical/lxd-pkg-snap/blob/latest-edge/snapcraft.yaml#L1442-L1448
If yes, I think I'd still keep the general stripping of binaries (https://github.com/canonical/lxd-pkg-snap/blob/latest-edge/snapcraft.yaml#L1591-L1600) just in case.